examples/relayer-signer-auth-v2/index.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+
+const { Defender } = require('@openzeppelin/defender-sdk');
+
+async function main() {
+  const client = new Defender({
+    relayerApiKey: process.env.RELAYER_API_KEY,
+    relayerApiSecret: process.env.RELAYER_API_SECRET,
+    authV2: true,
```

Comment:
Maybe `useCredentialsCaching` instead of `authV2`? 

Trying to find some better naming, but it is tricky

Reply:
Why not do this a default behavior and not have this toggle? Just being cautious?

Reply:
Yes, just in case. We will stress test the feature after next release, so if works we can make it default. But if fails for some reason, just setting the flag to false should fix it